WearTilesKotlin/app/build.gradle

```diff
@@ -63,16 +63,11 @@ android {
 }
 
 dependencies {
-    implementation libs.androidx.activity.compose
+    implementation libs.androidx.lifecycle.runtime.ktx
```

Comment:
This was a transitive dependency of the activity compose library but since we don't have any activities any more, I added the lifecycle dependency directly